core/lib/wallet_creator/src/main.rs

```diff
@@ -0,0 +1,70 @@
+use std::{env};
+
+use zksync::{
+    web3::{
+        types::{H160, H256},
+    },
+    zksync_types::{
+        tx::PackedEthSignature,
+    },
+    Network, RpcProvider, Wallet, WalletCredentials,
+};
+
+use zksync_eth_signer::{PrivateKeySigner};
+
+fn eth_random_account_credentials() -> (H160, H256) {
+    let mut eth_private_key = H256::default();
+    eth_private_key.randomize();
+
+    let address_from_pk = PackedEthSignature::address_from_private_key(&eth_private_key).unwrap();
+
+    (address_from_pk, eth_private_key)
+}
+
+async fn create_new_wallet(address: &str, key: &str) -> Result<Wallet<PrivateKeySigner, RpcProvider>, anyhow::Error>{
```

Comment:
I'd rename the second param to be clearer.
```suggestion
async fn create_new_wallet(address: &str, private_key: &str) -> Result<Wallet<PrivateKeySigner, RpcProvider>, anyhow::Error>{
```

Reply:
Updated